sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetryInfo.java

```diff
@@ -115,7 +115,7 @@ public String getMachineId() {
         return machineId;
     }
 
-    public void setMachineId(String machineId) {
+    public void setVmId(String vmId) {
         this.machineId = "vmId:" + machineId;
```

Comment:
This is a bug - `this.machineId = "vmId:" + machineId;` will only set the same machineId to machineId :)
Should be `this.machineId = "vmId:" + vmId;`:)

Reply:
True :-)